### ISSUE TITLE
processing / stroke() alpha

### ISSUE THREAD

**Issue by pinkfoot** [skulpt/skulpt#747]
Although `fill` alpha works, `stroke` alpha doesn't seem to have any effect - is this intended?

**Comment by dsblank** [comment]
No, that was an oversight. Would you like to try to fix this issue?

**Comment by pinkfoot** [comment]
I'm not yet familiar with Skulpt, but I think this one's very simple?  It passed the tests, so pr submitted...
